test/matchers/dimension_validator_matcher_test.rb

```diff
@@ -143,4 +143,62 @@ class ActiveStorageValidations::Matchers::DimensionValidatorMatcher::Test < Acti
     matcher.width_min 800
     refute matcher.matches?(Project.new)
   end
+
+  # width_min and height_min combined
+  test 'both width_min and height_min on higher combined are a positive match' do
+    matcher = ActiveStorageValidations::Matchers::DimensionValidatorMatcher.new(:dimension_min)
+    matcher.width_min 800
+    matcher.height_min 600
+    assert matcher.matches?(Project)
+  end
+
+  test 'width_min less than lower and height_min on higher combined are a negative match' do
+    matcher = ActiveStorageValidations::Matchers::DimensionValidatorMatcher.new(:dimension_min)
+    matcher.width_min 700
+    matcher.height_min 600
+    refute matcher.matches?(Project)
```

Comment:
Just because I saw it above - shouldn't this be `Project.new`?

Reply:
Well, I have followed the same test pattern as the other tests. Here we are testing the test matcher, if we pass a class (`Project`) to the `matches?` method, it will convert it to `Project.new` automatically in the method.